prototool.yaml

```diff
@@ -0,0 +1,123 @@
+      - REQUEST_RESPONSE_TYPES_UNIQUE                         # Verifies that all request and response types are unique to each RPC.
+      - RPCS_HAVE_COMMENTS                                    # Verifies that all rpcs have a comment of the form "// RPCName ...".
+      - RPCS_HAVE_SENTENCE_COMMENTS                           # Verifies that all rpcs have a comment that contains at least one complete sentence.
+      # - RPCS_NO_STREAMING                                     # Verifies that all rpcs are unary.
+      - RPC_NAMES_CAMEL_CASE                                  # Verifies that all RPC names are CamelCase.
+      - RPC_NAMES_CAPITALIZED                                 # Verifies that all RPC names are Capitalized.
+      # - RPC_OPTIONS_NO_GOOGLE_API_HTTP                        # Verifies that the RPC option "google.api.http" is not used.
+      - SERVICES_HAVE_COMMENTS                                # Verifies that all services have a comment of the form "// ServiceName ...".
+      - SERVICES_HAVE_SENTENCE_COMMENTS                       # Verifies that all services have a comment that contains at least one complete sentence.
+      # - SERVICE_NAMES_API_SUFFIX                              # Verifies that all service names end with "API".
+      - SERVICE_NAMES_CAMEL_CASE                              # Verifies that all service names are CamelCase.
+      - SERVICE_NAMES_CAPITALIZED                             # Verifies that all service names are Capitalized.
+      # - SERVICE_NAMES_MATCH_FILE_NAME                         # Verifies that there is one service per file and the file name is service_name_lower_snake_case.proto.
+      # - SERVICE_NAMES_NO_PLURALS                              # Verifies that all CamelCase service names do not contain plural components.
+      - SYNTAX_PROTO3                                         # Verifies that the syntax is proto3.
+      - WKT_DIRECTLY_IMPORTED                                 # Verifies that the Well-Known Types are directly imported using "google/protobuf/" as the base of the import.
+      # - WKT_DURATION_SUFFIX                                   # Verifies that all google.protobuf.Duration field names are "duration" or end in "_duration".
+      # - WKT_TIMESTAMP_SUFFIX                                  # Verifies that all google.protobuf.Timestamp field names are "time" or end in "_time".
+
+generate:
+  go_options:
+    import_path: github.com/Percona-Lab/percona-version-service/version
+
+  plugins:
+    - name: go
+      type: go
+      flags: plugins=grpc
+      output: ./versionpb
+
+    # - name: govalidators
+    #   type: go
+    #   flags: gogoimport=false
+    #   output: .
+
+    - name: grpc-gateway
+      type: go
+      flags: logtostderr=true
+      output: ./versionpb
+
+    - name: openapiv2
```

Comment:
Why you don't use github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger ? It integrates much better with grpc-gateway

Reply:
Well, there is no smart answer on that question.
As long as I had not worked with grpc-gateway before I used this repo as example https://github.com/johanbrandhorst/grpc-gateway-boilerplate
And my choice was based on that commit https://github.com/johanbrandhorst/grpc-gateway-boilerplate/commit/71ddff1d17279bdfd687bd48d9b028140bebf7ec . My suggestion was that openapiv2 was more newer thing to use rather than protoc-gen-swagger

Reply:
That commit is called "Update to grpc-gateway v2". You are using grpc-gateway v1 :)

Reply:
There will be no smart answer again :)
I just followed https://github.com/johanbrandhorst and used the last version